github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-presubmits.yaml

```diff
@@ -106,16 +103,16 @@ presubmits:
             - "-ce"
             - |
               cd images
-              ./publish_multiarch_image.sh -b bootstrap quay.io kubevirtci
-              ./publish_multiarch_image.sh -b -l golang quay.io kubevirtci
+              ./publish_image.sh -b bootstrap quay.io kubevirtci
+              ./publish_image.sh -b golang quay.io kubevirtci
           # docker-in-docker needs privileged mode
           securityContext:
             privileged: true
           resources:
             requests:
-              memory: "8Gi"
+              memory: "1Gi"
```

Comment:
maybe worth giving it bit more ?

Reply:
Hey @oshoval  - for all of the above - I am just returning the presubmit and postsubmit jobs back to the state before the multiarch build was introduced - https://github.com/kubevirt/project-infra/commit/fa4d2de19ce9fe0e8f67f5a66c1f58ae9ad5ac4a

Reply:
yep understood, just wondered if it wont timeout, because we usually have timeout defines
If we are safe, all good